std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
+  {
+    import std.file;
+    import std.path;
```

Comment:
It works because `unaryFun` imports everything and the kitchen sink. Imports at the call-site are irrelevant. It's quite the abomination...